README.md

```diff
@@ -1,5 +1,5 @@
 # Teiserver
-[![Build and test status](https://github.com/Beyond-All-Reason/teiserver/actions/workflows/elixir.yml/badge.svg?branch=master)](https://github.com/Beyond-All-Reason/teiserver/actions/workflows/elixir.yml/badge.svg?branch=master)
+[![Build and test](https://github.com/beyond-all-reason/teiserver/actions/workflows/tests.yml/badge.svg)](https://github.com/beyond-all-reason/teiserver/actions/workflows/tests.yml)
```

Comment:
Don't you want to keep the filter for the master branch in the link? It seems the new link is: https://github.com/beyond-all-reason/teiserver/actions/workflows/tests.yml?query=branch%3Amaster

Reply:
Fixed, used the link github generated.
https://github.com/beyond-all-reason/teiserver/actions/workflows/tests.yml/badge.svg?branch=master